src/app/rpmostree-builtin-status.c

```diff
@@ -422,10 +422,10 @@ status_generic (RPMOSTreeSysroot *sysroot_proxy,
         print_packages ("RemovedBasePackages", max_key_len,
                         origin_removed_base_packages, NULL);
 
-      /* let's be nice and only print requested - layered, rather than repeating
-       * the ones in layered twice */
-      if (origin_requested_packages)
-        print_packages ("RequestedPackages", max_key_len,
+      /* only print inactive layering requests in verbose mode */
+      if (origin_requested_packages && opt_verbose)
```

Comment:
Let's print a message then when adding an inactive request?  With this change then there's nothing visibly changed without `status -v`.